### PR TITLE
Add accessibility modifiers to the Entry Widget

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
https://glia.atlassian.net/browse/MOB-3697

**What was solved?**
Group accessibility media items. Add accessibility hints and types of items.

**Release notes:**

 - [ ] Feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

- [ ] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [ ] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to **Logging from iOS SDKs** → **Things to consider for newly added logs** in Confluence for more information.

**Screenshots:**

### CODE REVIEW

File: GliaWidgets/Sources/EntryWidget/EntryWidgetView.swift [hunk 1]
```diff
@@ -56,7 +55,9 @@ private extension EntryWidgetView {
                 isPlaceholder: true
             )
             .redacted(reason: .placeholder)
-            .disabled(true)
+            .accessibility(hidden: false)
+            .accessibilityElement(children: .combine)
+            .accessibilityLabel(Localization.EntryWidget.Loading.Accessibility.label)
```
Comment (EgorovEI) [inline GliaWidgets/Sources/EntryWidget/EntryWidgetView.swift:60]: To keep consistency we need to keep Accessibility values as a part of Style

Comment (igorkravchenko) [reply GliaWidgets/Sources/EntryWidget/EntryWidgetView.swift:60]: @EgorovEI Good catch.

**Review by igorkravchenko — COMMENTED** [review]
Apart from Egor mentioned, looks good to me.

**Review by igorkravchenko — APPROVED** [review]
@EgorovEI @rasmustautsglia  Just for clarity: localization is regenerated via swiftgen, that is why it is updated.

**Review by EgorovEI — APPROVED** [review]
LGTM. Please fix linter warnings in changed files

**Pull request merged**